pkgs/tools/games/minecraft/pacmc/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, stdenv
+, fetchzip
+, jre
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pacmc";
+  version = "0.5.2";
+
+  src = fetchzip {
+    url = "https://github.com/jakobkmar/pacmc/releases/download/${version}/pacmc-${version}.tar";
+    sha256 = "06qskp4n9ik3pya8j61zsi0dz0h2wx1r8w9qldlyp41qrl5g8zl7";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
```

Comment:
Add the missing pre/post `runHook`s. See https://nixos.org/manual/nixpkgs/unstable/#sec-stdenv-phases.